docs/updating/upgrading_to_v3.md

```diff
@@ -0,0 +1,570 @@
+
+2\. The deprecated Symfony Service definition `ezplatform.installer.db_based_installer` has been removed in favor of its FQCN-named definition.
+
+**Use:**
+
+``` php
+services:
+    Acme\App\Installer\MyCustomInstaller:
+        parent: EzSystems\PlatformInstallerBundle\Installer\DbBasedInstaller
+```
+
+**instead of:**
+
+``` php
+services:
+    Acme\App\Installer\MyCustomInstaller:
+        parent: ezplatform.installer.db_based_installer
+```
+
+### Other changes
+
+#### User checker
+
+Add the user checker to firewall by adding the following line to `config/packages/security.yaml`:
+
+``` yaml hl_lines="5"
+security:
+    firewalls:
+        ezpublish_front:
+            # ...
+            user_checker: eZ\Publish\Core\MVC\Symfony\Security\UserChecker
+            # ...
+```
+
+#### Command
+
+If your custom commands use `Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand`
+you need to rewrite them to use `Symfony\Component\Console\Command\Command` instead,
+because `ContainerAwareCommand` is deprecated.
+
+#### Permission-related methods
+
+Some deprecated [permission-related methods](../releases/ez_platform_v3.0_deprecations.md#permission-related-methods) have been removed.
+If your code uses them, you need to rewrite it to use the permission resolver.
+
+#### Container parameters
+
+A numer of Symfony Dependency Injection Container parameters [have been dropped](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/bc/8.0/dropped-container-parameters.md).
+
+To check if your code uses them, search for all occurrences of `ezpublish\..*\.class` (regular expression pattern)
+and [decorate Symfony services](https://symfony.com/doc/4.4/service_container/service_decoration.html) instead.
+
+#### QueryTypes
+
+If your code relies on automatically registering QueryTypes through the naming convention `<Bundle>\QueryType\*QueryType`,
+you need to register your QueryTypes as services and tag them with `ezpublish.query`, or enable their automatic configuration (`autoconfigure: true`).
+
+#### Symfony namespaces
+
+The following Symfony namespaces have changed. You need to update your code if it uses any of them:
+
+|Use|Instead of|
+|---|---|
+|Symfony\Contracts\Translation\TranslatorInterface|Symfony\Component\Translation\TranslatorInterface|
+|Symfony\Contracts\EventDispatcher\Event|Symfony\Component\EventDispatcher\Event|
+
+#### Deprecations
+
+For a full list of changes, see [Deprecations and backwards compatibility breaks](../releases/ez_platform_v3.0_deprecations.md)
+
+### Update the database
+
+Apply the following database update script:
+
+`mysql -u <username> -p <password> <database_name> < vendor/ezsystems/ezpublish-kernel/blob/master/data/update/mysql/dbupdate-7.5.latest-to-8.0.0.sql`
+
+or for PostgreSQL:
+
+`psql <database_name> < vendor/ezsystems/ezpublish-kernel/blob/master/data/update/postgres/dbupdate-7.5.latest-to-8.0.0.sql`
+
+!!! enterprise
+
```

Comment:
And for Enterprise they are here: https://github.com/ezsystems/ezplatform-ee/tree/master/upgrade/db

so maybe it's not worth keeping them here?